test/source/tests/setup.ts

```diff
@@ -507,7 +518,7 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       await settingsPage.waitAll(['@container-overlay-prompt-text', '@action-overlay-retry']);
       const renderedErr = await settingsPage.read('@container-overlay-prompt-text');
       expect(renderedErr).to.contain(`Failed to submit to Attester`);
-      expect(renderedErr).to.contain(`Could not find LDAP pubkey on a LDAP-only domain for email no.pub@client-configuration-test.flowcrypt.test on server keys.flowcrypt.test`);
+      expect(renderedErr).to.contain(`Your organization requires public keys to be present on company LDAP server, but no public key was found. Please ask your internal help desk.`);
```

Comment:
Please update to add ... no public key was found **for email address XXX**.

Reply:
In this situation(no keys found), don't we need to update `Failed to submit to Attester` error message like above?

> In this exact situation where we cross check against LDAP, the error should instead say `Attempting to import unknown key`

Reply:
Yes, also here